install/kubernetes/helm/istio/charts/tracing/templates/service.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
     chart: {{ template "tracing.chart" . }}
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
+  annotations:
+    networking.istio.io/exportTo: "."
```

Comment:
this should be set to * as all proxies send data to Zipkin service

Reply:
updated PTAL @rshriram